soroban-sdk/src/tests.rs

```diff
@@ -14,10 +14,8 @@ mod contract_store;
 mod contract_udt_enum;
 mod contract_udt_struct;
 mod contract_udt_struct_tuple;
-mod contractfile_with_sha256;
 mod contractimport;
 mod contractimport_with_error;
-mod contractimport_with_sha256;
```

Comment:
I removed these two tests because frankly they're a pain to keep updated. The sha256 is always changing. With this pull request they're impossible to keep updated because the repo builds the git commit into the wasm, therefore the sha256 cannot be known.